src/github.com/getlantern/flashlight/util/util_test.go

```diff
@@ -0,0 +1,162 @@
+package util
+
+import (
+	"crypto/x509"
+	"io/ioutil"
+	"net/http"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/getlantern/fronted"
+	"github.com/getlantern/keyman"
+	"github.com/mailgun/oxy/forward"
```

Comment:
`oxy/forward` package is not in repo, though it will definitely be added later by the chained server implementation.

`flashlight/util` is not added in `testpackages.txt`.